docs: update deleting vfolder with detailed guide (#64)
lablup/backend.ai-docs-webui: docs/vfolder/vfolder.rst

-------------

If you have permission to delete the storage folder, you can delete it by

> clicking the trash can icon in the Control pane.By clicking the icon, 

Reviewer: ```suggestion
clicking the trash can icon in the Control pane. By clicking the icon, 
```

it will automatically move the folder to trash bin with notification, 

.. image:: delete_notification.png
   :width: 400
   :align: center
   :alt: Folder delete notification modal

> and status is  changed to ``delete-pending`` status. 

Reviewer: ```suggestion
and status is changed to ``delete-pending`` status. 
```

.. image:: trash_bin_table.png
   :width: 800
   :align: center
   :alt: Trash bin table

In this status, you can restore the folder by
clicking restore button in Control pane. If you want to delete the folder forever, 
then please click red trash bin icon button. A confirmation modal will pop up with 
input field to make sure you want to delete this folder completely. After typing the 

> name of folder correctly, click ``DELETE FOREVER`` button to remove

Reviewer: ```suggestion
name of folder correctly, click ``DELETE FOREVER`` button to remove.
```

-------------

If you have permission to delete the storage folder, you can delete it by

> clicking the 'trash can' icon in the control column. By clicking the icon, 

Reviewer: ```suggestion
clicking the 'trash can' button in the control column. By clicking the icon, 
```

Author: https://github.com/lablup/backend.ai-docs-webui/pull/64#issuecomment-2149091268 Same in here.

the system will automatically move the folder to 'trash can' with following notification, 

.. image:: delete_notification.png
   :width: 400
   :align: center
   :alt: Folder delete notification modal

and the folder status appearing in the 'status' column will be changed to ``delete-pending``. 

.. image:: trash_bin_table.png
   :width: 800
   :align: center
   :alt: Trash bin table

In this status, you can restore the folder by

> clicking restore button in Control pane. If you want to permanently delete the folder, 

Reviewer: ```suggestion
clicking 'restore' button in Control column. If you want to permanently delete the folder, 
```

> please click 'red trash can' icon in the same column. A confirmation modal will pop up with 

Reviewer: ```suggestion
please click 'red trash can' button in the same column. A confirmation modal will pop up with 
```

Author: https://github.com/lablup/backend.ai-docs-webui/pull/64#issuecomment-2149091268 Same here.
